### ISSUE TITLE
Disabling output file extension

### ISSUE THREAD

**Issue by burt202** [kubb-labs/kubb#1422]
### What version of `kubb` is running?

3.0.10

### What kind of platform do you use?

Linux

### How does your `kubb.config.ts` config look like

```typescript
See linked repo below
```

### Swagger/OpenAPI file?

```bash
See linked repo below
```

### What version of external packages are you using(`@tanstack-query`, `MSW`, `React`, `Vue`, ...)

```JSON
See linked repo below
```

### What steps can reproduce the bug?

In v2, the contents of the `index.ts` file is `export * from "./types";` but in v3 it adds `.ts` by default. I can get around this by doing:
```
extension: {
  ".ts": ""
}
```

but `""` is not a valid to the `KubbFile.Extname` type. Is there a better way of doing this or does the typing need to be relaxed to allow empty string to disable it?

For repro, clone repo linked below and `npm ci` and `npm run generate`

https://github.com/burt202/kubb-test

Cheers, hopefully this is the last issue to enable us to upgrade to v3 smoothly!

### How often does this bug happen?

Every time

**Comment by stijnvanhulle** [comment]
Fixed with v`3.0.11`

**Comment by burt202** [comment]
Yep, `3.0.11` is all good with
```
extension: {
  ".ts": ""
}
```
Cheers!
